statscache/templates/stats.html

```diff
@@ -0,0 +1,284 @@
+{% extends "layout.html" %}
+{% block title %}{{ plugin.name.lower() }}{% endblock %}
+{% block byline %}direct feed{% endblock %}
+{% block head %}
+    <meta name="description" content="Raw feed of '{{ plugin.ident }}' model">
+    <link rel="stylesheet"
+          type="text/css"
+          href="{{ url_for('static', filename='style/dashboard.css') }}" >
+    </link>
+    <link rel="stylesheet"
+          type="text/css"
+          href="{{ url_for('static', filename='style/stats.css') }}" >
+    </link>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js"></script>
```

Comment:
We use local files to prevent tracking of our users that some cdn tend to do.

We add the version number to the file to know which version we're at and thus can track updates :)